src/net.h

```diff
@@ -74,6 +74,7 @@ struct mg_connection {
   unsigned is_draining : 1;    // Send remaining data, then close and free
   unsigned is_closing : 1;     // Close and free the connection immediately
   unsigned is_full : 1;        // Stop reads, until cleared
+  unsigned is_throttled : 1;   // Last TLS write: MG_SOCK_PENDING() was true
```

Comment:
Bad name, too generic. Something like `is_tls_remaining_data` - make up a good name!

Reply:
Mongoose is being throttled by the transport layer, the fact that we only care in TLS is secondary... for other cases MG_IO_WAIT because there is no intermediate buffer nor encryption, so we bypass setting this flag as it has no use.
`is_tls_throttled` ?